postingest/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "postingest_table_name" {
+  value = local.postingest_state_table_name
+}
+
+output "postingest_table_arn" {
+  value = module.postingest_state_table.table_arn
+}
+
+output "postingest_confirmer_queue_arn" {
+  value = module.dr2_custodial_copy_confirmer_queue.sqs_arn
+}
```

Comment:
You'll need to output the lambda arn here as well and then add it to the list in `deploy_lambda_policy.tf` The deploy job is failing at the moment.

Reply:
You've got it in the output, you just need to add the output to the deploy role.
So add `module.postingest.postingest_state_change_lambda_arn` to the bottom of the list here 
https://github.com/nationalarchives/dr2-terraform-environments/blob/main/deploy_roles.tf#L51

Reply:
Oops, I missed the 2nd half of the sentence.